src/styles/main.less

```diff
@@ -44,21 +54,32 @@ body {
   font-family: 'Teko';
   font-weight: 300;
   text-transform: uppercase;
+  text-rendering: optimizeLegibility;
+  letter-spacing: .2px;
   font-size: 20px;
   height: 70px;
   margin: 0;
+  border-radius: 0;
   .navbar-nav {
     li {
       margin: 18px 2px 0 0;
       a {
         padding: 6px 12px;
         &:hover {
           border-radius: 4px;
+          color: @navbar-default-bg;
+          background-color: @nav-link-hover-bg - 50%;
```

Comment:
Please use vars like navbar-default-link-color - no need to redefine them explicitly